manifests/default.pp

```diff
@@ -109,5 +103,5 @@
 
 service { 'openfisca':
     ensure  => 'running',
-    require => [ File['/etc/init/openfisca.conf'], Python::Pip['gunicorn'] ],
+    require => [ File['/etc/init/openfisca.conf'], Exec['fetch openfisca requirements'] ],
```

Comment:
As discussed IRL, that's the name given to `pip install --upgrade -r openfisca/requirements.txt` done in the correct virtualenv.